app/koi-client/src/page/@backoffice@screen@[partyId]/component/StockScreen/Table.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
 const Table = ({ stockId }: Props) => {
   const { data: stock, timeIdx } = Query.Stock.useQueryStock(stockId, { keepPreviousData: false });
 
-  if (!stock?.companies || !timeIdx) {
+  if (!stock?.companies || timeIdx === -1) {
```

Comment:
```suggestion
  if (!stock?.companies || timeIdx === undefined) {
```

!timeIdx는 0이라는 숫자도 포함하기 때문에, 수정하는 것은 적절했습니다.
하지만, -1 이라는 숫자는 어떤 의도인지 알기 어렵습니다.
잘못된 값을 명확하게 표기하기 위해서는 `undefined`가 적절해 보입니다.

Reply:
위와 동일합니다!

Reply:
오홋 넵 undefined; 로 변경해 뒀습니당!